### PR TITLE
layout-search-bar

### PR DESCRIPTION
**Pull request by svyatoslavPy**
- [DEMO LINK](https://svyatoslavPy.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://svyatoslavPy.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,27 @@
...
+
+    <!-- Search bar -->
+    <div class="search-bar__content">
+      <div class="search-bar">
+        <input
+        class="search-bar__input
+        search-bar__input--size--large"
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles”">
```
Comment (l4st1m0za) [inline src/index.html:23]:
```suggestion
        <input
          class="search-bar__input
          search-bar__input--size--large"
          type="text"
          data-qa="big"
          placeholder="Try “Los Angeles”"
        >
```

Use this styling for all the cases.

**Review by l4st1m0za — CHANGES_REQUESTED** [review]
Grerat job! One thing to change is your font weight. In the tests it's clear that you have some difference with the needed result!

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <!-- Search bar -->
+    <form
+      class="search-bar__form"
+      data-qa="keypress"
+      method="post">
```
Comment (anastasiiavorobiova) [inline src/index.html:19]:
```suggestion
    <form
      class="search-bar__form"
      data-qa="keypress"
      method="post"
    >
```

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,30 @@
...
+      <div class="search-bar">
+        <input
+          class="search-bar__input
+          search-bar__input--size--large"
+          type="text"
+          data-qa="big"
+          placeholder="Try “Los Angeles”">
```
Comment (anastasiiavorobiova) [inline src/index.html:26]:
```suggestion
          placeholder="Try “Los Angeles”"
        >
```

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,30 @@
...
+        <span class="search-bar__icon search-bar__icon--large"></span>
+      </div>
+      <div class="search-bar">
+        <input
+          class="search-bar__input search-bar__input--size--small"
+          type="text"
+          data-qa="small"
+          placeholder="Try “Los Angeles”">
```
Comment (anastasiiavorobiova) [inline src/index.html:34]:
```suggestion
          placeholder="Try “Los Angeles”"
        >
```

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**